app/lib/send-email.js

```diff
@@ -3,8 +3,17 @@ const { applicationEmailDocRequestMsgType, applicationdDocCreationRequestQueue }
 const { carbonCopyEmailAddress, templateIdFarmerClaimComplete } = require('../config').notify
 const sendMessage = require('../messaging/send-message')
 const appInsights = require('applicationinsights')
+const sendSFDEmail = require('./sfd-client')
+const { sfdMessage } = require('../config')
 
 const send = async (templateId, email, personalisation) => {
+  if (sfdMessage.enabled) {
```

Comment:
I can see that you've tried to hew close to the original implementation here, perhaps to avoid having to stray too far - but I envisaged us moving away from this, perhaps abstracting one level up, rather than intertwining at this level. We're no longer sending an email here, we are sending a request message. 
In this app we should be moving away from the current logic that is making multiple requests out to send email to individual addresses, as that exists due to the limitation of the gov.notify API being hit directly here - since we are abstracted away from that via new route, we should not be be repeating it just because it existed before. From here we should be sending a single request to the queue and ultimately the SFD proxy. The contract between here and there should allow us to specify multiple email addresses and we should utilise that.

Reply:
I think for practical purposes we'll have to just stick with this for now, even though I think it can be done better. We will revisit to remedy that later, as long as we can get it working is the main thing.